prov/rxm/src/rxm_cq.c

```diff
@@ -1297,6 +1297,13 @@ void rxm_ep_do_progress(struct util_ep *util_ep)
 	while (!dlist_empty(&rxm_ep->repost_ready_list)) {
 		dlist_pop_front(&rxm_ep->repost_ready_list, struct rxm_rx_buf,
 				buf, repost_entry);
+
+		/* Discard rx buffer if its msg_ep was closed */
+		if (!buf->conn->msg_ep) {
+			ofi_buf_free(&buf->hdr);
```

Comment:
Hmm, all checkers failed. Looks like for 1.7.x we should use
`util_buf_release(rxm_ep->buf_pools[RXM_BUF_POOL_RX].pool, buf);` 
instead of
`ofi_buf_free(&buf->hdr); `

Reply:
thanks -updated and pushed